### PR TITLE
Made page have responsive nav bar, spacing everywhere else is broken

### PR DESCRIPTION
**Pull request by TurnbulA**
Added a responsive nav bar 
Spacing everywhere else is broken will fix in separate pr

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
+html {
+	font-family: 'Poppins', sans-serif;
+	font-size:10px;
+
 }
+
 body {
 	background-color: #F7F8F8;
+	padding-bottom: 5rem;
+	font-size:1.6rem;
```
Comment (mali891) [inline style.css]: Space after colon or no space? Pick one and run with it. Tip: we use space after colon at YW.

File: style.css [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
+	overflow-x:hidden;
+	max-width: 100%;
+}
+h1{
+    font-size:auto;
```
Comment (mali891) [inline style.css]: not necessary

File: style.css [hunk 3]
```diff
@@ -126,15 +98,14 @@ body {
...
 	justify-content: center;
 	position: relative;
-	z-index: 0;
 	margin-top: -24rem;
 }
 .header-image {
 	width: 80%;
 	max-height: 50rem;
 }
 .button-border {
```
Comment (mali891) [inline style.css:104]: Space after selector or no space? Pick one and run with it. Tip: we use a space after selector at YW.

File: style.css (hunk 1 above)
Comment (TurnbulA) [reply style.css]: Gone through and changed all spacing to space after colon

File: style.css (hunk 2 above)
Comment (TurnbulA) [reply style.css]: Removed h1 tag

**Review by mali891 — APPROVED** [review]
Good stuff

**Pull request merged**